src/app/content/components/popUp/Filters.tsx

```diff
@@ -73,13 +76,19 @@ const Toggle = styled(React.forwardRef<HTMLButtonElement, ToggleProps>(
   }
 `;
 
+interface FilterDropdownProps extends ControlledProps {
```

Comment:
pretty sure there is a helper to get the prop types out of a component so you could extend that instead of exporting the props interface separately

Reply:
If `Dropdown` component would be a normal component then we could use `React.ComponentProps<typeof Dropdown>` but it is a `styled` component so this basically becomes "accept anything that normal HTML element can"

I've added
```ts
type TabTransparentDropdownProps = CommonDropdownProps & Props;
export type TabHiddenDropdownProps = CommonDropdownProps & (Props | Props & ControlledProps);
```
to `Dropdown.tsx` and used `TabHiddenDropdownProps` for this